packages/blade/src/components/Input/SelectInput/SelectInput.stories.tsx

```diff
@@ -0,0 +1,15 @@
+import type { Meta } from '@storybook/react';
+import SelectInputComponent from './SelectInput';
+
+const SelectInputStoryMeta: Meta = {
```

Comment:
probably, that's because the dropdown is not implemented yet.

also, one thing to notice here is that maybe we need to focus the select input when the icon is clicked? right now it might be confusing because there's no feedback and hence probably why divyanshu is also confused

Reply:
yes I think we should ignore pointer events on trailing icons from BaseInput. It's not working even after implementing dropdown

Reply:
it should work now. Treating it as an interactiveElement instead of trailingIcon.

Reply:
https://github.com/razorpay/blade/pull/932/commits/d592ea33d6fe21f83da579994c56a6567f00899d